frontend/src/types.ts

```diff
@@ -20,6 +23,14 @@ export const NIP_47_SIGN_MESSAGE_METHOD = "sign_message";
 
 export const NIP_47_NOTIFICATIONS_PERMISSION = "notifications";
 
+export const NIP_47_PAY_KEYSEND_METHOD = "pay_keysend";
+export const NIP_47_MULTI_PAY_KEYSEND_METHOD = "multi_pay_keysend";
+export const NIP_47_MULTI_PAY_INVOICE_METHOD = "multi_pay_invoice";
+
+export const SCOPE_GROUP_FULL_ACCESS = "full_access";
```

Comment:
We should try to only put types here which are used in multiple files.

Also, there is a mix of these constant variables, and string unions. We should be consistent here